lib/matplotlib/pyplot.py

```diff
@@ -1515,20 +1529,32 @@ def subplot_mosaic(mosaic, *, sharex=False, sharey=False,
         behave as for `subplots`.  If False, each subplot's x- or y-axis will
         be independent.
 
-    subplot_kw : dict, optional
-        Dictionary with keywords passed to the `.Figure.add_subplot` call
-        used to create each subplot.
+    width_ratios : array-like of length *ncols*, optional
+        Defines the relative widths of the columns. Each column gets a
+        relative width of ``width_ratios[i] / sum(width_ratios)``.
+        If not given, all columns will have the same width.
+        *gridspec_kw* can take this as a key, and takes precedence.
```

Comment:
does it still take precedence if the error message means either it's present as a kwarg or a dictionary entry?

Reply:
Ooops, thanks for catching that. I changed in the 4 places this was incorrect....